docs/api-specs/reporting-api-v1.yaml

```diff
@@ -4,7 +4,7 @@ info:
   description: |-
     API to manage reports that provide data calculated based on any entities of the Aam Digital system
     and offer notifications when data of such reports changes.
-  version: 1.0.0-alpha.4
+  version: 1.0.0-alpha.5
```

Comment:
maybe we should have an official version 1.0 by now (it is used in production, after all) and have these changes as a 1.1-beta ?